runelite-client/src/main/java/net/runelite/client/plugins/nightmarezone/NightmareZoneConfig.java

```diff
@@ -109,11 +110,22 @@ default boolean absorptionNotification()
 		return true;
 	}
 
+	@ConfigItem(
+			keyName = "hitpointsnotification",
+			name = "1 HP notification",
+			description = "Sends a notification if your hitpoints goes above 1",
```

Comment:
"Toggles notifications when your absorption points gets below your threshold" is the message for absorption potions. Could you be consistent with this?